torch/_inductor/kernel/flex/flex_attention.py

```diff
@@ -50,6 +50,19 @@
 aten = torch.ops.aten
 Expr = sympy.Expr
 
+ForceImpl: TypeAlias = Literal["DEFAULT", "DECODE", "FLASH", "TRITON"]
```

Comment:
You can import a type alias from another file. If torch compile doesn't like it, we should add a comment to that effect